src/Gallery/Components/GalleryFileExtensions.ts

```diff
@@ -1,4 +1,6 @@
 export const GalleryFileExtensions = {
   image: ['jpg', 'jpeg', 'png', 'webp'],
   svg: ['svg'],
+  doc: ['pdf'],
```

Comment:
Should we include doc, docx, xls and xlsx here to support word and excel? For the card we could just show a word or excel icon.